e2e/test/Helpers/ConsoleEventListener.cs

```diff
@@ -1,89 +1,50 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
 
 namespace System.Diagnostics.Tracing
 {
     public sealed class ConsoleEventListener : EventListener
     {
-        private readonly string[] _eventFilters;
+        private static readonly string[] _eventFilters = new string[] { "DotNetty-Default", "Microsoft-Azure-Devices" };
```

Comment:
One suggestion is to allow the console logger to be initialized with a list of event providers. The reason for this is, we recommend customers use this class to help them debug their test code, and they wouldn't want all of our sdk relevant event sources to be written by default.

Reply:
Doesn't that assume customers log to ETW too?

Reply:
Here is what I found on testing. OnEventSourceCreated get called even before we populate the _eventFilters in the constructor. It was extremely odd but happened. Hence, I removed that. It is possible to miss subscribing to some if we go that route. It becomes misleading. I am open to other suggestions.